packages/blocks/src/surface-block/managers/layer-manager.ts

```diff
@@ -528,19 +541,15 @@ export class LayerManager {
       insertToOrderedArray(array, element);
     };
 
+    if (shouldUpdateGroupChildren) {
+      this._init();
```

Comment:
This seems to be a `_reset()` method? This runs more than once during the lifecycle of `LayerManager`.